### ISSUE TITLE
vagrant rsync doesn't work after the machine is provisioned

### ISSUE THREAD

**Issue by jfrumar** [displague/vagrant-linode#40]
When provisioning a machine, the rsync operation completes, but in order to continue development we need to repeatedly rsync our updated code.

When running via virtualbox, a `sudo vagrant rsync` will push our files across to the vm. Unfortunately this doesn't work with the linode provider.

The weird part is that it works when initially provisioning the machine - it just doesn't work afterwards.

**Comment by displague** [comment]
I believe this has been corrected in v0.2.1.

**Comment by displague** [comment]
```
displague@d:~/src/vagrant-linode$ vagrant rsync prod4-newark
==> prod4-newark: Machine State ID: active
==> prod4-newark: IP Address: 45.79.167.228
==> prod4-newark: Machine State ID: active
==> prod4-newark: IP Address: 45.79.167.228
==> prod4-newark: Machine State ID: active
==> prod4-newark: IP Address: 45.79.167.228
==> prod4-newark: Rsyncing folder: /home/displague/src/vagrant-linode/ => /vagrant
==> prod4-newark:   - Exclude: [".vagrant/", ".git/", "vendor/"]
```
